controllers/controllers.go

```diff
@@ -49,10 +49,24 @@ func (r *instanceReconciler) configureInstance(instance *instances.InstanceInfo,
 	if err != nil {
 		return errors.Wrap(err, "failed to create new nodeconfig")
 	}
-	if err := nc.Configure(); err != nil {
-		return errors.Wrap(err, "failed to configure Windows instance")
+
+	// if instance does not have an associated node, configure it
+	if instance.Node == nil {
+		return nc.Configure()
 	}
-	return nil
+
+	// Version annotation equal to the current version means that the node has been fully configured by this version
+	// of WMCO. No further action is required.
```

Comment:
This again seems to be a change in current behavior. Previously we would have reconfigured the `Machine` instance irrespective of the version annotation. Why are you doing this?

Reply:
In practice are never doing that. In the case that the annotation has the incorrect value [we immediately attempt to delete the machine](https://github.com/openshift/windows-machine-config-operator/blob/master/controllers/windowsmachine_controller.go#L253-L271).

This function is only called by the WindowsMachine controller if the version annotation is not present